app.js

```diff
@@ -86,3 +97,41 @@ console.log('token: ' + token);
 http.createServer(app).listen(app.get('port'), function () {
   console.log('Express server listening on port ' + app.get('port'));
 });
+
+exports.create = function (req, res, next) {
```

Comment:
### <sub><img src="https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-medium.svg" width="20"/></sub>&nbsp; Allocation of Resources Without Limits or Throttling

This endpoint handler performs a system command execution and does not use a rate-limiting mechanism. It may enable the attackers to perform Denial-of-service attacks. Consider using a rate-limiting middleware such as express-limit.

<h6>
<a href="https://cwe.mitre.org/data/definitions/770.html">CWE-770</a> | <a href="https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score">Priority score 582</a> | Line 101</h6>